packages/connector-local/src/connector.ts

```diff
@@ -298,7 +300,11 @@ export default class LocalConnector implements IConnector {
             } as any));
         }
 
-        await Promise.all(files.map(this.fetch.bind(this)));
+        try {
+            await Promise.all(files.map(this.fetch.bind(this)));
+        } catch (e) {
+            console.error(e);
```

Comment:
we should use `logger` instead of console.

Reply:
That's a leftover that shouldn't be in there.